sequence.go

```diff
@@ -92,3 +93,75 @@ func getFeatureSequence(feature Feature, location Location) string {
 
 	return sequenceString
 }
+
+// the following functions Iter and iter, derive from github.com/schwarmco/go-cartesian-product
+// which uses interfaces, so I modified it to use runes
+func Iter(params ...[]rune) chan []rune {
+	// create channel
+	c := make(chan []rune)
+	// create waitgroup
+	var wg sync.WaitGroup
+	// call iterator
+	wg.Add(1)
+	iterate(&wg, c, []rune{}, params...)
+	// call channel-closing go-func
+	go func() { wg.Wait(); close(c) }()
+	// return channel
+	return c
+}
+
+// private, recursive Iteration-Function
+func iterate(wg *sync.WaitGroup, channel chan []rune, result []rune, params ...[]rune) {
+	// dec WaitGroup when finished
+	defer wg.Done()
+	// no more params left?
+	if len(params) == 0 {
+		// send result to channel
+		channel <- result
+		return
+	}
+	// shift first param
+	p, params := params[0], params[1:]
+	// iterate over it
+	for i := 0; i < len(p); i++ {
+		// inc WaitGroup
+		wg.Add(1)
+		// create copy of result
+		resultCopy := append([]rune{}, result...)
+		// call self with remaining params
+		go iterate(wg, channel, append(resultCopy, p[i]), params...)
+	}
+}
+
+func allVariantsIUPAC(seq string) []string {
```

Comment:
Make public (capitalize `allVariantsIUPAC` -> `AllVariantsIUPAC` ) and create export comment starting as `// AllVariantsIUPAC (insert description here)`

Reply:
@TimothyStiles Does the export comment go on the same line as the function declaration?